README.md

```diff
@@ -844,6 +862,29 @@ If a notification token could not be generated on the device (e.g. due to missin
 
 None.
 
+### shareHealthSummary
+
+`shareHealthSummary` creates a share code for an unauthenticated user to export a health summary using alternative credentials.
+
+#### Security
+
+This function may be called by admins (for any patient), owners/clinicians (for patients of the same organization), or patients (for themselves).
+
+#### Input
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|userId|optional string|-|The patient's user id. If not provided or null, the requesting user's userId is used instead.|
+
+#### Output
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|code|string|-|An 8 character alphanumeric code. It is only checked case-insensitively, so you may change its casing when displaying it to the user.|
+|expiresAt|Date|-|The date the share code expires at. Exporting a health summary with an expired share code will fail. Currently, the expiration date is 5 minutes after requesting the share code.|
```

Comment:
@pauljohanneskraft I've seen the function actually returning a date 10 minutes after the current time, not 5.

Reply:
I changed it to 5 minutes now - and also removed the fullUrl property.